lib/active_admin/views/components/status_tag.rb

```diff
@@ -60,7 +53,7 @@ def convert_to_boolean_status(status)
         case status
         when true, 'true'
           'Yes'
-        when false, 'false', nil
+        when *FALSE_VALUES
```

Comment:
@deivid-rodriguez I don't know. I tried to see if I made the change on my own or if there was a comment in #3862 by someone to suggest otherwise but doesn't seem to be. I must have done that on my own, but don't remember. I can reset it back to what it was. Should I do that?